test/ability_compound_eyes.c

```diff
@@ -3,7 +3,7 @@
 
 SINGLE_BATTLE_TEST("Compound Eyes raises accuracy")
 {
-    PASSES_RANDOMLY(91, 100);
+    PASSES_RANDOMLY(91, 100, WRNG_ACCURACY);
```

Comment:
Somehow this is easier for me to parse as "X out of Y", but maybe we could consider percentage here as well?
```
PASSES_RANDOMLY(20, WEIGHTED_RNG_ACCURACY);
```
would say "20% of the tests need to pass.